DOCS-2541: Move component APIs (#3504)
viamrobotics/docs: docs/appendix/apis/components/generic.md

---
title: "Control your generic component with the generic API"
linkTitle: "Generic"
weight: 20
type: "docs"

> description: "The generic API allows you to give commands to your generic components for running model-specific commands using DoCommand."

Reviewer: ```suggestion
description: "Commands for running custom model-specific commands using DoCommand on your generic components."
```

> description: "The generic API allows you to give commands to your generic components with code instead of with the graphical interface of the Viam app"

Reviewer: Whatever you end up doing with these descriptions, I think they should end with periods :D
